src/ShadowsOfShadows.csproj

```diff
@@ -31,19 +31,28 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Microsoft.CSharp" />
-    <Reference Include="SadConsole">
+    <Reference Include="MonoGame.Framework, Version=3.5.1.1679, Culture=neutral, processorArchitecture=MSIL">
+      <HintPath>packages\MonoGame.Framework.DesktopGL.3.5.1.1679\lib\net40\MonoGame.Framework.dll</HintPath>
```

Comment:
Trzeba ręcznie usunąć te zmiany dot. wersji w `Include=` bo to nam ogranicza aplikację. Jak tego nie ma, to można kompilować z dowolną wersją biblioteki.